bin/compile

```diff
@@ -63,24 +68,17 @@ BLACKFIRE_AGENT_SOCKET=unix://${BF_AGENT_SOCKET_FILE}
 # ----------
 # Install the Blackfire agent and CLI tool
 # ----------
-topic "Installing the latest version of Blackfire agent and CLI client"
-curl -s -o agent.tar.gz -L https://blackfire.io/api/v1/releases/agent/linux/amd64
-tar -xzf agent.tar.gz
-chmod +x agent
-cp agent ${BF_BUILD_BIN_DIR}/blackfire-agent
-rm agent.tar.gz agent agent.sha1
-
-echo "Installing blackfire CLI client" | indent
-curl -A "Heroku" -o blackfire_cli.tar.gz -s -L https://blackfire.io/api/v1/releases/client/linux/amd64
-tar -xzf blackfire_cli.tar.gz
+topic "Installing the latest version of Blackfire CLI"
+curl -s -o blackfire.tar.gz -L https://blackfire.io/api/v1/releases/cli/linux/amd64
+tar -xzf blackfire.tar.gz
 chmod +x blackfire
 cp blackfire ${BF_BUILD_BIN_DIR}/blackfire
-rm blackfire_cli.tar.gz blackfire blackfire.sha1
+rm blackfire.tar.gz blackfire blackfire.sha1
 
-# Configure the agent and the client.
+## Configure the Agent and the Client.
 topic "Blackfire configuration"
 echo "Configuring the agent" | indent
-$BF_BUILD_BIN_DIR/blackfire-agent --register \
+$BF_BUILD_BIN_DIR/blackfire agent:config \
```

Comment:
I think it makes sense to use `-q` here, otherwise, a long message, not really applicable to the Heroku environment (restart instructions etc) are printed:

```
remote: Checking credentials
remote: Writing configuration to /tmp/build_8aa5effc/.blackfire/etc/agent
remote: 
remote: ------------------------------------------------------------------------------
remote: The configuration file has been updated successfully
remote: 
remote: /tmp/build_8aa5effc/.blackfire/etc/agent
remote: 
remote: Thank you for using Blackfire
remote: ------------------------------------------------------------------------------
remote: Please now restart the blackfire-agent service to use the generated configuration
remote: sudo service blackfire-agent restart
```

But not sure what happens if the credentials are wrong...

Reply:
Ah, it says:

```
remote: The Blackfire API answered with a 401 HTTP error (Invalid credentials.)
```

That might be enough, given how the buildpack already checks if the env vars are all there etc.